### DISCUSSION TITLE
Suggestion/question re additional data that could be conveyed

### DISCUSSION THREAD

**Discussion by gavinsboyle** [sly9/VoyagerTelegramBot#27] [Q&A]
Hi guys
may as well start the discussion a bit...
it might be helpful before we users start lobbing in all kind of requests to understand the art of the possible a bit.
by way of example, it would be neat (and maybe even useful) if i could send the already generated/updated pdf of the light curve from my photometry pipeline as well as the thumbnail of the last image? or is it not as simple as just giving another file path-name in the config file and an additional push line in the core code?
best
gavin

**Comment by bigpizza** [comment]
Hi Gavin,

Glad to see you here 😃 Sending an additional file from somewhere on the hard drive is possible, but there are still some issues in my personal opinion that should be discussed before actually put it in product. For example, when requesting a file from personal folders, modern OS like Mac OS and Windows 11 requires more previlidges. Some of our users are sensetive to such thing. Also, in your case, the additional data is generated by other pipeline so in general, the VoyagerTelegramBot does not know when is the proper timing to send the new data.

We can definitely discuss the possibility and the best practice to make it work.

Thanks,

Kun
